cli_ui/__init__.py

```diff
@@ -145,43 +148,35 @@ def tuple(self) -> Tuple[Token, ...]:
         return (self.as_string,)
 
 
-def using_colorama() -> bool:
-    if os.name == "nt":
-        if "TERM" not in os.environ:
-            return True
-        if os.environ["TERM"] == "cygwin":
-            return True
-        return False
-    else:
-        return False
-
-
 def config_color(fileobj: FileObj) -> bool:
+    # if CONFIG["color"] has been set, i.e, different
+    # from "auto", use it:
     if CONFIG["color"] == "never":
         return False
     if CONFIG["color"] == "always":
         return True
-    if os.name == "nt":
-        # sys.isatty() is False on mintty, so
-        # let there be colors by default. (when running on windows,
-        # people can use --color=never)
-        # Note that on Windows, when run from cmd.exe,
-        # console.init() does the right thing if sys.stdout is redirected
+    # CONFIG["color"] is auto, but maybe color support has
+    # been set in the environment - see
+    # https://bixense.com/clicolors/ for details
+    if os.environ.get("CLICOLORS") == "0":
+        return False
+    if os.environ.get("CLICOLORS_FORCE", "0") != "0":
         return True
```

Comment:
According to the standard, if CLICOLORS_FORCE is enabled, you must enable colors. This condition should be checked first.